hebrewreader.py

```diff
@@ -95,17 +93,17 @@ def fix_gloss(gloss):
         return 'I'
     return re.sub(r'<(.*)>', r'\\textit{\1}', gloss)
 
-def get_passage_and_words(passage, api, separate_chapters=True, verse_nos=True):
+def get_passage_and_words(passage, api, lang, separate_chapters=True, verse_nos=True):
```

Comment:
Could you make the order `lang, api, passage`? (I see now that the old order is incorrect, too.)